### PR TITLE
Update cscope to 15.9 and add allow symlinks patch

### PR DESCRIPTION
**Pull request by nwg**
* Removed old patch that added APPLE TERMINFO check (upstream added)
* Added patch to allow symlinks in cscope database build and search
  (file not found error for all symlinks)

Tested on
macOS 10.15.2 19C57
Xcode 11.0 11A420a

### CODE REVIEW

File: devel/cscope/Portfile [hunk 1]
```diff
@@ -1,7 +1,7 @@
 PortSystem 1.0
 
 name                cscope
-version             15.8a
+version             15.9
 revision            1
```
Comment (mf2k) [inline devel/cscope/Portfile]: The `revision` line needs to be reset to 0 when upgrading the `version`.

**Comment by mf2k** [comment]
Your commit messages need to squashed into 1. You also need to add a reference to the Trac ticket you created:

`Closes: https://trac.macports.org/ticket/59857
`

**Comment by nwg** [comment]
@mf2k how do i resolve the change request? I have squashed the commits and updated the message to point to the ticket.

**Comment by mf2k** [comment]
> @mf2k how do i resolve the change request? I have squashed the commits and updated the message to point to the ticket.

You don't because only the requester can. 

You still have not fixed all of the issues:

- The revision line still needs to be fixed
- The commit message needs to start with the port name.

`cscope: ...
`

**Pull request merged**